docs/GovernanceModel.md

```diff
@@ -6,16 +6,24 @@ is to enable a vibrant development community for the Open Enclave SDK. If you
 feel we should make any changes to our guidelines, please start a discussion
 with us.
 
-Our model is based on the [liberal contribution
-policy](https://opensource.guide/leadership-and-governance/). We make decisions
-through a consensus seeking process, rather than a formal voting process.
+Our model is based on the
+[liberal contribution policy](https://opensource.guide/leadership-and-governance/).
+We make most decisions through a consensus seeking process, rather than a formal
+voting process. This means that contributions will be accepted once no
+committers object (and our PR guidelines have been met). If there is a major
```

Comment:
As phrased, this is problematic since it means you cannot accept a contribution unless you know that 0 objections are raised, so you have to slow down accepting contributions (e.g., PRs) until everyone has a chance to review, and have everyone as a reviewer on every PR, which I don't think it practical.

Also the "no committers object" contradicts Maintainers.md lines 10-14 which is better written.  That text is much better.  This text here is, in my opinion, in need of changes.

Reply:
I don't think this says or reads as "every committer needs to review every PR." I think our processes for properly reviewing PRs is (a) separate from this framework and (b) under improvement (see #1920). A committer objecting is a committer reviewing a PR and actually objecting; I don't think this reads a non-review as an objection.

How would you rephrase this to make that more clear?

Reply:
Just got it re-worded. Let me know what you think.